### PR TITLE
add (dummy) dispatch.dropExports to liveslots, test that it can be called

### LINKED ISSUE #2653: add a no-op dispatch.dropExports to the kernel-to-vat pathway
## What is the Problem Being Solved?

The second preparation step for #2615 is to provide a `dispatch.dropExports()` call to all vats. It doesn't need to do anything yet, but as long as it exists, we can proceed with kernel-side work that might emit `dropExports` without fear of crashing upon a missing method.

The signature will be `dispatch.dropExports(vrefs)`, taking an array of vat-side object references. We won't support dropping promises for now.

This needs to be added to both liveslots, the comms vat, and the vat workers (if they look closely enough at the vat/kernel protocol to notice the individual messages being sent).

## Description of the Design

A later step will make `dropExports` do real work: it needs to delete a strong reference from the planned `exports` Set, and remove slotToVal/valToSlot entries for the vref, so that liveslots is no longer keeping the exported Remoteable alive. (It might be kept alive by other local references, but not liveslots).

When the kernel calls `dropExports`, it promises to never mention those identifiers again, at least until the vat re-exports one of them by mentioning it in some subsequent syscall. As a result, it would be an error for the kernel to `dropExports` the same vref twice (without an intervening re-export).

## Test Plan

`test-liveslots.js` will be augmented to invoke `dropExports` and make sure it doesn't crash.

### PR DESCRIPTION
**Pull request by warner**
The new `dropExports()` doesn't do anything yet, but at least test that it
can be called directly in liveslots. It should be reachable through all vat
worker types, but I won't be able to test that until we have a kernel
mechanism to provoke it.

closes #2653
refs #2615

### CODE REVIEW

**Comment by warner** [comment]
This new version (613c165) adds `dropExports` to the comms vat too. We need it in both comms and liveslots to allow kernel-side work to proceed without causing "no such method" problems.

**Comment by warner** [comment]
Oops, I have one more commit to add to this, I forgot to wire `dropExports` into the local vat manager too.

Also, it's failing because of some XS thing.. which I thought I made optional, weird.

**Review by FUDCo — APPROVED** [review]
Aside from the repeated boilerplate in all the vat workers, which we already know is a more general issue, this looks fine to me.

**Comment by warner** [comment]
There's a small overlap between this and #2680, so I'm going to hold off landing it until that one lands.

**Pull request merged**
